src/components/SearchForm.js

```diff
@@ -1,31 +0,0 @@
-import React from "react";
-import PropTypes from "prop-types";
-import "../styles/SearchForm.css";
-
-function SearchForm({ searchText, setSearchText, onSubmit }) {
-  const handleInputChange = (event) => {
-    setSearchText(event.target.value);
-  };
-
-  return (
-    <div className="search-form">
-      <input
-        className="searchInput"
-        type="text"
```

Comment:
consider adding a placeholder to help the user work out what needs to be typed. For example:
`placeholder="London`
or
`placeholder="city`